src/styles/radio.scss

```diff
@@ -60,18 +54,18 @@ $block: #{$fd-namespace}-radio;
     display: flex;
     align-items: center;
     overflow: hidden;
-    padding: $fd-radio-outer-circle-margin;
+    padding: var(--fdRadio_Outer_Circle_Padding);
 
     &::before {
       content: "";
-      height: $fd-radio-inner-circle-diameter;
-      width: $fd-radio-inner-circle-diameter;
-      min-width: $fd-radio-inner-circle-diameter;
+      height: var(--fdRadio_Inner_Circle_Diameter);
+      width: var(--fdRadio_Inner_Circle_Diameter);
+      min-width: var(--fdRadio_Inner_Circle_Diameter);
```

Comment:
this block of code is repeated 4 times. Maybe it's a good idea to create a mixin and use it in checkbox and radio button code.